.gitignore

```diff
@@ -44,5 +44,3 @@ yarn-error.log
 # VSCode workspace settings
 .vscode
 
-# ts outDir
-lib/
```

Comment:
I think lib should not live in the repository, it gets added in the workflow through `yarn build` before deploy to the NPM registry

Reply:
Yes
You are right about that but until the PR is merged I am using these changes as a fork. Now since it's a fork the yarn build does not work and therefore I had to push the lib folder.
Can you provide any suggestions on how to proceed with this?

Reply:
Use a different branch containing these changes instead of using your PR?